Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Service_Ownership.yml

```diff
@@ -71,37 +71,6 @@ tasks:
               field:
                 value:
                   simple: id
-          filters:
```

Comment:
Does your script still not need to check for `alert.asmsystemids.type` have the value `ASSET-NAME` or `ASSET-ID`?

Reply:
Discussed: `ASSET-ID` is not needed. `ASSET-NAME` is used for the scripts business logic optionally.